src/lib/containers/provenance/ProvenanceGraph.tsx

```diff
@@ -375,6 +375,15 @@ const ProvenanceReactFlow = (props: ProvenanceProps) => {
     onEdgesChangedListener,
   ])
 
+  const onPaneScrollFunction: (event?: React.WheelEvent) => void = useCallback(
+    event => {
+      if (event && event.deltaY && rootEntityRefs.length > 0) {
+        window.scrollTo(window.scrollX, window.scrollY + event?.deltaY)
+      }
+    },
+    [rootEntityRefs],
```

Comment:
So I artificially depended on rootEntityRefs

Reply:
If this truly does not depend on rootEntityRefs, does it work if you remove ` && rootEntityRefs.length > 0` and set the dependencies to `[]`? It doesn't seem like this function needs to be re-created, but maybe I'm misunderstanding.

Alternatively, you could define the function outside of the component.

Reply:
Removed dependency.  This solution works _pretty_ well.  If you try to scroll inside of a node, it still eats the event.  So there still might be some frustration when attempting to scroll past the graph, but with this change it's now likely that you'll be able to scroll to where you want to be (instead of accidentally zooming in/out the graph).